src/components/Home.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Country from './Country';
+import './Home.css';
+import load from '../assets/loading.gif';
+
+function Home() {
+  const data = useSelector((state) => state.countries);
+  return (
+    <div className="home_container">
+
```

Comment:
- [ ] It would be better if you implement the filter functionality as mentioned by the previous code reviewer. Kindly find below is a screenshot of the requirements
![Screenshot from 2022-04-16 11-15-43](https://user-images.githubusercontent.com/46388113/163673004-1c8314c9-8703-48fa-ba2c-99b4ff6de4d3.png)